extensions/dependencies/guppy.html

```diff
@@ -1,8 +1,8 @@
-<link rel="stylesheet" href="/third_party/static/guppy-f6e0bb/build/guppy.min.css">
-<script src="/third_party/static/guppy-f6e0bb/build/guppy.min.js"></script>
+<link rel="stylesheet" href="/third_party/static/guppy-260da66/build/guppy.min.css">
```

Comment:
Any reason to use 7-char hash rather than 6-char hash? Not that it matters too much either way, except that we've always been using 6-char hash so this breaks convention.